### PR TITLE
[Training-Project] POC to use OpenCV to count vehicles and sent them to CEP

### PR DESCRIPTION
**Pull request by hemikak**
The following is a CEP sample for $subject.

### CODE REVIEW

File: modules/samples/consumers/object-detection-service/pom.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (suhothayan) [inline modules/samples/consumers/object-detection-service/pom.xml]: Licenses header is missing

File: modules/samples/consumers/object-detection-service/src/main/webapp/index.jsp [hunk 2]
```diff
@@ -0,0 +1,133 @@
+<html>
```
Comment (suhothayan) [inline modules/samples/consumers/object-detection-service/src/main/webapp/index.jsp]: Licenses header is missing

File: modules/samples/producers/object-detection-client/pom.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (suhothayan) [inline modules/samples/producers/object-detection-client/pom.xml]: Licenses header is missing

File: modules/samples/producers/video-frame-client/pom.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (suhothayan) [inline modules/samples/producers/video-frame-client/pom.xml]: Licenses header is missing

File: modules/samples/producers/video-frame-client/src/main/resources/log4j.properties [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (suhothayan) [inline modules/samples/producers/video-frame-client/src/main/resources/log4j.properties]: Change this to WSO2 Licenses header

File: modules/samples/consumers/object-detection-service/src/main/java/org/wso2/carbon/sample/objectdetection/server/DetectedImageDisplay.java [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    public void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        String event = request.getParameter("event");
+        if (event.equals("display")) {
+            TreeMap<Integer, StreamData> streamDataMap = MapHelper.getStreamDataTreeMap();
+            if (!streamDataMap.isEmpty()) {
+                StreamData streamData = streamDataMap.firstEntry().getValue();
+                Gson gson = new Gson();
+                JsonElement element = gson.toJsonTree(streamData, new TypeToken<StreamData>() {
+                }.getType());
```
Comment (hastef88) [inline modules/samples/consumers/object-detection-service/src/main/java/org/wso2/carbon/sample/objectdetection/server/DetectedImageDisplay.java]: weird formatting

File: modules/samples/consumers/object-detection-service/pom.xml (hunk 1 above)
Comment (hemikak) [inline modules/samples/consumers/object-detection-service/pom.xml]: Fixed in https://github.com/hemikak/product-cep/commit/6210d26b510acc5fc699d74e24cdcd17d3c8dd1d

File: modules/samples/consumers/object-detection-service/src/main/webapp/index.jsp (hunk 2 above)
Comment (hemikak) [inline modules/samples/consumers/object-detection-service/src/main/webapp/index.jsp]: Fixed in https://github.com/hemikak/product-cep/commit/6210d26b510acc5fc699d74e24cdcd17d3c8dd1d

File: modules/samples/producers/object-detection-client/pom.xml (hunk 3 above)
Comment (hemikak) [inline modules/samples/producers/object-detection-client/pom.xml]: Fixed in https://github.com/hemikak/product-cep/commit/6210d26b510acc5fc699d74e24cdcd17d3c8dd1d

File: modules/samples/producers/video-frame-client/pom.xml (hunk 4 above)
Comment (hemikak) [inline modules/samples/producers/video-frame-client/pom.xml]: Fixed in https://github.com/hemikak/product-cep/commit/6210d26b510acc5fc699d74e24cdcd17d3c8dd1d

File: modules/samples/producers/video-frame-client/src/main/resources/log4j.properties (hunk 5 above)
Comment (hemikak) [inline modules/samples/producers/video-frame-client/src/main/resources/log4j.properties]: Changed in https://github.com/hemikak/product-cep/commit/6210d26b510acc5fc699d74e24cdcd17d3c8dd1d

**Pull request closed without merging**